cli_reference/get_started_cli.adoc

```diff
@@ -113,6 +113,17 @@ directory on your PATH. To check your PATH, open the Command Prompt and run:
 C:\> path
 ----
 
+ifdef::openshift-origin,openshift-enterprise[]
+[NOTE]
+====
+The `oc` command provides command line tools for managing applications, while
+the `oadm` command provides tools for managing a cluster. The `openshift` binary
+is an all-in-one command that can be used for starting services that make up the
+{product-title} cluster. The `openshift` binary can perform all the same actions
+as the `oc` and `oadm` commands.
+====
+endif::[]
+
```

Comment:
2 confusions:
one is I'm not sure why adding this paragraph in `get_started_cli.adoc`. This seems to just duplicate the above adding in `admin_cli_operations.adoc`?
the other is why adding this common document between `For Windows` and `For Mac OS X`?